### ISSUE TITLE
Deselect via the dropdown

### ISSUE THREAD

**Issue by mweghorst** [angular-ui/ui-select#1651]
When using a multiple select dropdown with the remove-selected option set to false, shouldn't the selected options in the dropdown list be clickable and deselect their value, instead of being disabled? This is the same behaviour as in the original select2, see https://select2.github.io/examples.html (Multiple select boxes).

**Comment by user378230** [comment]
Can you rephrase as an feature request please?

**Comment by AndreasIllg** [comment]
Did you work on this? I need this feature as well and hacked my way around it by setting "isTag=true" for each choice (this avoids disabling, you can select it multiple times now) and removing an item if it is selected for the second time (in the "on-select" callback, you can inject $selectMultiple and call $selectMultiple.removeChoice for both).
But that's a hack I would like to get rid of soon. It also produces a flickering since the item appears a second time and is removed immediately.
So is there any progress on this feature?

**Comment by user378230** [comment]
@AndreasIllg feel free to submit a PR for this feature 😃

**Comment by wellaya** [comment]
good feature to have
